practice-one/src/js/models/model.js

```diff
@@ -36,6 +35,57 @@ class BookModel {
             return [];
         }
     };
+
+  deleteBookByInfo = (bookInfo) => {
+    const savedBooks = storage.get("savedBooks");
+    if (savedBooks) {
+      const bookIndex = savedBooks.findIndex(
+        (book) =>
+          book.bookname === bookInfo.bookname &&
```

Comment:
How about with this case: we have 2 items the same bookname, author, date, description? which item will be remove/delete